Packs/ThreatGrid/Integrations/ThreatGrid/ThreatGrid.py

```diff
@@ -872,14 +867,14 @@ def append_to_analysis_iocs_arrays(iocs, dbots, k):
     """
     Helper for appending analysis item to ioc an dbot arrays
     """
-    demisto.info(f"check k {k}")
-    iocs.append(ioc_to_readable(k))
-    dbots.append({
-        'Vendor': 'ThreatGrid',
-        'Type': 'IOC',
-        'Indicator': k.get('ioc', 'default'),
-        'Score': calc_score(k.get('severity', 'default'))
-    })
+    if k and k.get('ioc'):
+        iocs.append(ioc_to_readable(k))
+        dbots.append({
+            'Vendor': 'ThreatGrid',
+            'Type': 'IOC',
+            'Indicator': k.get('ioc'),
+            'Score': calc_score(k.get('severity', '1'))
```

Comment:
So that means that if there is no severity we will mark the indicator as good, correct? 
I think this should be marked as Unknown in such a case.
I think the calc_score method needs to be changed accordingly. 
![image](https://user-images.githubusercontent.com/50294648/117274490-c5e54900-ae65-11eb-9d2e-5252fc319a44.png)